app.json

```diff
@@ -8,13 +8,7 @@
     "icon": "./assets/icon.png",
     "userInterfaceStyle": "light",
     "plugins": [
-      [
-        "expo-camera",
-        {
-          "cameraPermission": "Allow R|P 2024 to access your camera",
```

Comment:
do we not need the camera permission anymore?

Reply:
Oh not sure why that is removed. I can add that back in.

Reply:
not sure why that was removed. I don't think I touched that. will add it again.